a4kSubtitles/search.py

```diff
@@ -227,22 +227,63 @@ def __prepare_results(core, meta, results):
                 color_list = group
                 break
 
+    def _filter_name(x):
+        name_diff_ignore = quality + codec + audio + color
+        name_diff_ignore += ["multi", 'multiple', 'sub', 'subs', 'subtitle']
+
+        if x.isdigit():
+            x = str(int(x)).zfill(3)
+        elif x.lower() in name_diff_ignore:
+            x = ''
+        return x.lower()
+
+    def _match_numbers(a, b):
+        offset = 0
+        for s in b:
+            s = core.re.sub(r'v[1-4]', "", s)
```

Comment:
Re-release versions: if translators re-release fixing or improving or so of the same episode may also adds "v2" or "v3" e.g. "Series name - 100v2" 

This line to avoid those cases.